app/mailers/user_mailer.rb

```diff
@@ -20,4 +20,13 @@ def task_changed
     @url  = "http://localhost:3000/tickets/#{@new_ticket.id}"
     mail(to: @user.email, subject: 'Your ticket has been changed')
   end
+
+
+  def ping_person
+    @user = params[:user]
+    @comment = params[:comment]
+    @url1  = "http://localhost:3000/tickets/#{@comment.ticket_id}"
```

Comment:
https://guides.rubyonrails.org/action_mailer_basics.html#generating-urls-in-action-mailer-views